cms/celery.py

```diff
@@ -34,6 +34,5 @@ def alternate_env_tasks(self):
         return {
             'openedx.core.djangoapps.content.block_structure.tasks.update_course_in_cache': 'lms',
             'openedx.core.djangoapps.content.block_structure.tasks.update_course_in_cache_v2': 'lms',
-            'openedx.core.djangoapps.grades.tasks.compute_grades_for_course': 'lms',
-            'openedx.core.djangoapps.grades.tasks.compute_grades_for_course_v2': 'lms',
+            'lms.djangoapps.grades.tasks.compute_all_grades_for_course': 'lms',
         }
```

Comment:
This is getting pulled by my PR too, that might be what's causing you conflicts. Will try to merge today.